[2025-10-09T04:51:19.043+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: order_generator.load_orders scheduled__2025-10-07T15:00:00+00:00 [queued]>
[2025-10-09T04:51:19.057+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: order_generator.load_orders scheduled__2025-10-07T15:00:00+00:00 [queued]>
[2025-10-09T04:51:19.064+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2025-10-09T04:51:19.066+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 2
[2025-10-09T04:51:19.068+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2025-10-09T04:51:19.095+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): load_orders> on 2025-10-07 15:00:00+00:00
[2025-10-09T04:51:19.108+0000] {standard_task_runner.py:55} INFO - Started process 2435 to run task
[2025-10-09T04:51:19.112+0000] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'order_generator', 'load_orders', 'scheduled__2025-10-07T15:00:00+00:00', '--job-id', '120', '--raw', '--subdir', 'DAGS_FOLDER/order_loader_dag.py', '--cfg-path', '/tmp/tmpm_mgs4xx']
[2025-10-09T04:51:19.115+0000] {standard_task_runner.py:83} INFO - Job 120: Subtask load_orders
[2025-10-09T04:51:19.239+0000] {task_command.py:389} INFO - Running <TaskInstance: order_generator.load_orders scheduled__2025-10-07T15:00:00+00:00 [running]> on host 81fbc710bbce
[2025-10-09T04:51:19.338+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=order_generator
AIRFLOW_CTX_TASK_ID=load_orders
AIRFLOW_CTX_EXECUTION_DATE=2025-10-07T15:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-10-07T15:00:00+00:00
[2025-10-09T04:51:19.382+0000] {logging_mixin.py:137} INFO - Successfully loaded data to orders
[2025-10-09T04:51:19.621+0000] {logging_mixin.py:137} INFO - Successfully loaded data to order_items
[2025-10-09T04:51:19.622+0000] {python.py:177} INFO - Done. Returned value was: None
[2025-10-09T04:51:19.632+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=order_generator, task_id=load_orders, execution_date=20251007T150000, start_date=20251009T045119, end_date=20251009T045119
[2025-10-09T04:51:19.687+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2025-10-09T04:51:19.705+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
